app/src/main/AndroidManifest.xml

```diff
@@ -240,6 +240,10 @@
         <activity android:name=".onboarding.SuggestedEditsOnboardingActivity" />
 
         <activity android:name=".suggestededits.SuggestedEditsImageTagsOnboardingActivity" />
+        <activity
+            android:name=".suggestededits.SuggestedEditsFeedCardImageTagActivity"
+            android:launchMode="singleTask"
```

Comment:
What's the reasoning for making this `singleTask`?

Reply:
Done